greenwave/resources.py

```diff
@@ -305,15 +305,15 @@ def retrieve_scm_from_koji_build(nvr: str, source: str, koji_url: str):
 @cached
 def retrieve_yaml_remote_rule(url: str):
     """ Retrieve a remote rule file content from the git web UI. """
-    response = requests_session.request('HEAD', url)
+    response = requests_session.request('GET', url)
+
     if response.status_code == 404:
         log.debug('Remote rule not found: %s', url)
         return None
 
     if response.status_code != 200:
```

Comment:
```suggestion
_raise_for_status(response)
```
will do the same thing

Reply:
That would require more changes since it throws different type of exception (`requests.exceptions.HTTPError`).

Reply:
I didn't mean a built-in function but `_raise_for_status` with an underscore in the beginning, defined in the same file (line 54). It throws the BadGateway exception.

Reply:
OK, reverted to the `_raise_for_status(response)`.